src/Css.elm

```diff
@@ -369,9 +409,36 @@ All CSS properties can have the values `unset`, `initial`, and `inherit`.
 @docs BoxShadowConfig, boxShadow, defaultBoxShadow
 
 
+## Border
+
+@docs border, border2, border3
+
+@docs borderTop, borderTop2, borderTop3
+
+@docs borderRight, borderRight2, borderRight3
+
+@docs borderBottom, borderBottom2, borderBottom3
+
+@docs borderLeft, borderLeft2, borderLeft3
+
+
+## Border Width
+
+@docs borderWidth, borderWidth2, borderWidth3, borderWidth4, borderTopWidth, borderRightWidth, borderBottomWidth, borderLeftWidth
+
+@docs thin, thick
+
+
 ## Border Style
 
-@docs wavy, dotted, dashed, solid, double, groove, ridge, inset, outset
+@docs borderStyle, borderStyle2, borderStyle3, borderStyle4, borderTopStyle, borderRightStyle, borderBottomStyle, borderLeftStyle
+
+@docs dotted, dashed, solid, double, groove, ridge, inset, outset
+
+
+## Border Color
+
+@docs borderColor, borderColor2, borderColor3, borderColor4, borderTopColor, borderRightColor, borderBottomColor, borderLeftColor
```

Comment:
I'm little bit worry about the docs structure but this kind of approach the most logical and clear by my opinion.

Reply:
This grouping seems fine to me! 👍